src/OpenLoco/src/Objects/ObjectStringTable.cpp

```diff
@@ -27,40 +27,40 @@ namespace OpenLoco::ObjectManager
     };
 
     constexpr std::array<uint8_t, 34> kNumStringsPerObjectType = {
-        1,
```

Comment:
just a bonus for this PR, copied from ObjectManager.h to make it more clear here which string count belongs to which object